### ISSUE TITLE
Pay for landscape after release mouse click

### ISSUE THREAD

**Issue by luk3ZPL** [OpenLoco/OpenLoco#1456]
**Describe the solution you'd like**
Current game behavior on the land adjustment: you paying when you click (hold it) and you move mouse (up/down)

**Describe alternatives you've considered**
Alternative game behavior on the land adjustment: you paying after releasing mouse button (up/down)

**Additional context**
Easy land adjustment. You will know what level you preffer without paying for it during mouse moving.
![land](https://user-images.githubusercontent.com/4522457/163034016-c297e3db-cd0d-4cba-81f8-e98363f3a71a.PNG)

**Comment by LeftofZen** [comment]
What happens if you move the land so that the total cost is more money than you have, then release the mouse button? Does the land go back to its original position?

**Comment by luk3ZPL** [comment]
> What happens if you move the land so that the total cost is more money than you have, then release the mouse button? Does the land go back to its original position?

Nope. you may check it yourself.  Currently I'm paying every time I move the mouse. Land is adjusting as high as I can afford for it. Dunno if it is trivial to implement, but you probably know how it is working now. This proposed behavior should be virtually drawed as projecting/prototyping, so you will pay only when you adjust it at desired level.

**Comment by LeftofZen** [comment]
Ok so you'd like something like a "landscape ghost", that functions the same as the track and vehicle ghosts?

**Comment by luk3ZPL** [comment]
> Ok so you'd like something like a "landscape ghost", that functions the same as the track and vehicle ghosts?

Yes something alike. Wouldn't it be more convenient ?
